examples/src/Form.tsx

```diff
@@ -0,0 +1,42 @@
+import * as React from 'react';
+import { connectForm, IFormProps } from '../../src/main';
+import * as styles from './form.module.css';
+import JsonView from 'react-pretty-json';
+import 'react-pretty-json/assets/json-view.css';
+import { isRequired } from '../../src/validators';
```

Comment:
Should we maybe move the validators to the example and not include them as part of the library?

Reply:
added https://github.com/vitkon/form-container/issues/34 for it